app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def markdown(text)
```

Comment:
@no-reply I couldn't find any specific exceptions from Redcarpet, but I added some comments to the helper method, added a test, and added a check for String input:  https://github.com/dpla/primary-source-sets/commit/aeefa533ae04b851d135e3a3609d073449aa7363